### PR TITLE
[no ticket][risk=no] e2e nightly test "owner-copy-to-workspace.spec.ts" is not working

### PR DESCRIPTION
**Pull request by aweng98**
Test failure was caused by code issue: `clickButton() fp.flow`.
I noticed test failure message today in CircleCI nightly [job](https://app.circleci.com/pipelines/github/all-of-us/workbench/11702/workflows/ccbe078d-2dba-4fc9-904b-e64b200f1575/jobs/149645/artifacts) test log. Test failed silently, so the CircleCI job did not fail.

```
BUILD SUCCESSFUL in 13s
20 actionable tasks: 1 executed, 19 up-to-date
Running owner-copy-to-workspace.spec at 7/9/2021, 00:16:19
Running gce-to-dataproc.spec at 7/9/2021, 00:16:19
Saved screenshot: logs/screenshot/Copy-notebook-to-another-Workspace-when-CDR-versions-match.png
Saved html file: logs/html/Copy-notebook-to-another-Workspace-when-CDR-versions-match.html
```

### CODE REVIEW

File: e2e/tests/nightly/owner-copy-to-workspace.spec.ts [hunk 1]
```diff
@@ -81,10 +78,23 @@ async function copyNotebookTest(sourceWorkspaceName: string, destCdrVersionName:
...
+  expect(
+    modalText.some((text) => {
+      return text === `Successfully copied ${sourceNotebookName} to ${destWorkspace}`;
+    })
+  ).toBeTruthy();
+  expect(
+    modalText.some((text) => {
+      return text === 'Do you want to view the copied Notebook?';
+    })
+  ).toBeTruthy();
```
Comment (aweng98) [inline e2e/tests/nightly/owner-copy-to-workspace.spec.ts:96]: Verify more modal texts.

File: e2e/app/container.ts [hunk 2]
```diff
@@ -63,18 +63,18 @@ export default class Container {
...
+          waitFn: () => {
+            this.page.waitForNavigation({ waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout });
           }
         },
         {
           shouldWait: waitForClose,
-          waitFn: async () => {
-            await this.waitUntilClose(timeout);
+          waitFn: () => {
+            this.waitUntilClose(timeout);
```
Comment (aweng98) [inline e2e/app/container.ts:77]: Revert this function back before commit [4993](https://github.com/all-of-us/workbench/pull/4993/files).

**Review by freemabd — COMMENTED** [review]
Seems that you have a test failure

**Comment by aweng98** [comment]
Rerun all tests. I haven't been able to figure out why it failed but it seems to be unrelated to PR changes.

**Comment by aweng98** [comment]
all tests passed this time.

**Review by freemabd — APPROVED** [review]
Looks good.

**Pull request merged**